app/screens/home/search/results/file_options/file_options.tsx

```diff
@@ -1,90 +1,139 @@
+        onActionComplete?.();
     }, [serverUrl, fileInfo.post_id, intl]);
 
-    return (
-        <View>
-            <Header fileInfo={fileInfo}/>
-            {canDownloadFiles &&
+    const optionItems = useMemo(() => {
+        return (
+            <>
+                {canDownloadFiles &&
                 <OptionItem
                     action={handleDownload}
                     label={intl.formatMessage({id: 'screen.search.results.file_options.download', defaultMessage: 'Download'})}
                     icon={'download-outline'}
                     type='default'
                 />
-            }
-            <OptionItem
-                action={handlePermalink}
-                label={intl.formatMessage({id: 'screen.search.results.file_options.open_in_channel', defaultMessage: 'Open in channel'})}
-                icon={'globe'}
-                type='default'
-            />
-            {enablePublicLink &&
+                }
+                <OptionItem
+                    action={handlePermalink}
+                    label={intl.formatMessage({id: 'screen.search.results.file_options.open_in_channel', defaultMessage: 'Open in channel'})}
+                    icon={'globe'}
+                    type='default'
+                />
+                {enablePublicLink &&
                 <OptionItem
                     action={handleCopyLink}
                     label={intl.formatMessage({id: 'screen.search.results.file_options.copy_link', defaultMessage: 'Copy link'})}
                     icon={'link-variant'}
                     type='default'
                 />
-            }
-            <View style={styles.toast} >
-                {action === 'downloading' &&
+                }
+            </>
+        );
+    }, [canDownloadFiles, enablePublicLink, handlePermalink, handleDownload, handleCopyLink]);
+
+    const tablet = useMemo(() => {
+        return (
+            <View
+                style={[styles.tablet, openUp ? styles.openUp : styles.openDown]}
+            >
+                {optionItems}
+            </View>
+        );
+    }, [optionItems, openUp]);
+
+    const mobile = useMemo(() => {
+        return (
+            <>
+                <Header fileInfo={fileInfo}/>
+                {optionItems}
+                <View style={styles.toast} >
+                    {action === 'downloading' &&
                     <DownloadWithAction
                         action={action}
                         item={galleryItem}
                         setAction={setAction}
                     />
-                }
-                {action === 'copying' &&
+                    }
+                    {action === 'copying' &&
                     <CopyPublicLink
                         item={galleryItem}
                         setAction={setAction}
                     />
-                }
-            </View>
-        </View>
-    );
+                    }
+                </View>
```

Comment:
This is a question for @matthewbirtch. I didn't add the toasts that show on mobile when an action completes on the tablet. The reason I didn't add them for tablet is because when a user clicks and action (download, copy link, go to channel), the menu is closed, indicating that it was completed.

Reply:
toasts are now added to tablet views also.